submissions/Lu-sure/Popup/style.css

```diff
@@ -0,0 +1,216 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  height: 65px;
+  align-items: center;
+}
+
+.nav-icon, .popup-list {
+  list-style: none;
+}
+
+.nav-icon {
+  display: flex;
+  align-items: center;
+}
+
+.nav-text-item {
+  text-decoration: none;
+  color: #5f5f5f;
+  margin-right: 25px;
+}
+
+.nav-text-item:hover, .popup-bottom:hover {
+  text-decoration: underline;
+}
+
+.nav-img-item, .menu-img {
+  height: 32px;
+  width: 32px;
+}
+
+.nav-img-item {
+  margin-right: 25px;
+}
+
+.menu-item-border {
+  border: 3px solid transparent;
+  border-radius: 8%;
+}
+
+.menu-item-border:hover {
+  border-color: rgb(145, 181, 248);
+}
+
+.bell-item {
+  width: 26px;
+  height: 30px;
+}
+
+.bell-img {
+  width: 100%;
+  height: 100%;
+}
+
+.round-img {
+  border-radius: 50%;
+}
+
+.hide-checkbox {
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way.